tests/e2e/specs/upe-split/shopper/shopper-deferred-intent-creation-upe-enabled.spec.js

```diff
@@ -11,16 +11,18 @@ import {
 	confirmCardAuthentication,
 	fillCardDetails,
 	setupProductCheckout,
+	selectGiropayOnCheckout,
+	completeGiropayPayment,
 } from '../../../utils/payments';
 import { uiUnblocked } from '@woocommerce/e2e-utils/build/page-utils';
 const { shopper, merchant } = require( '@woocommerce/e2e-utils' );
 
 const UPE_METHOD_CHECKBOXES = [
-	'#inspector-checkbox-control-3', // bancontact
-	'#inspector-checkbox-control-4', // eps
-	'#inspector-checkbox-control-5', // giropay
-	'#inspector-checkbox-control-6', // ideal
-	'#inspector-checkbox-control-7', // sofort
+	'#inspector-checkbox-control-5', // bancontact
+	'#inspector-checkbox-control-6', // eps
+	'#inspector-checkbox-control-7', // giropay
+	'#inspector-checkbox-control-8', // ideal
+	'#inspector-checkbox-control-9', // sofort
```

Comment:
One thing I'm not too clear here - why did the checkbox ids need to change? Did they need to change for some unrelated reasons to the other changes (adding the test case) in this PR?

Reply:
Sorry, I forgot to submit this comment - it's non-blocking

Reply:
Good question. As I noted in the PR description, they changed after the addition of Affirm and Afterpay.

Reply:
Ah, I missed that bullet point! Thanks for clarifying! :)